.circleci/config.yml

```diff
@@ -273,8 +273,12 @@ jobs:
           name: "Check TypeScript for @deriv/stores"
           command: npx tsc --project packages/stores/tsconfig.json -noEmit
       - run:
-          name: "Check TypeScript for @deriv/wallets"
-          command: npx tsc --project packages/wallets/tsconfig.json -noEmit
+          name: "Check TypeScript and linting for @deriv/wallets"
+          command: |
+            npx tsc --project packages/wallets/tsconfig.json -noEmit
+            npx eslint --fix --config packages/wallets/.eslintrc.js packages/wallets
+            npx stylelint packages/wallets/**/*.scss
```

Comment:
@adrienne-deriv Let's do them separately to have a more readable CI steps and logs 🙏🏻